tests/build_/test_arrangements.py

```diff
@@ -1040,3 +1049,54 @@ def test_inject_parent_image(self, osbs, build_type):
             'koji_hub': osbs.build_conf.get_kojihub()
         }
         assert args == expected_args
+
+    @pytest.mark.parametrize('scratch', [False, True])  # noqa:F811
+    def test_flatpak(self, osbs, scratch):
```

Comment:
Would also be nice to add a test to make sure those plugins are removed for non-flatpak arrangement-version-4 builds.

Reply:
Done